tests/unit/RetroGenFunction.unitTest.ts

```diff
@@ -64,21 +51,4 @@ describe("Retro Gen Function", () => {
       }
     });
   });
-
```

Comment:
why are we removing this?

Reply:
I removed this as it was testing that process record was throwing an error